config/StaticData.rb

```diff
@@ -23,6 +23,16 @@ class StaticData
 
   EXCEPTION_FILES = JSON.load_file("#{Dir.pwd}/config/exception_file.json")
 
+  INVALID_TOKEN_ERROR = '-8'
+
+  def self.documentserver_jwt_exist?
+    ENV.include?('DOCUMENTSERVER_JWT')
+  end
+
+  def self.documentserver_jwt_empty?
+    ENV.fetch('DOCUMENTSERVER_JWT', '') != ''
```

Comment:
```
ENV.fetch('DOCUMENTSERVER_JWT') != ''
```

And juding by name - it should check if key is empty

So correct check will be

```
ENV.fetch('DOCUMENTSERVER_JWT') == ''
```

Reply:
If the variable does not exist, an exception is raised.

https://apidock.com/ruby/Hash/fetch

So I use the default empty string to check the condition

Reply:
Fair enoght